content/blogs/release-0-19.md

```diff
@@ -0,0 +1,769 @@
+
+```yaml
+id: schedule_subflows
+namespace: company.team
+
+tasks:
+  - id: generate_dates
+    type: io.kestra.plugin.scripts.python.Script
+    beforeCommands:
+      - pip install pytz
+    script: |
+      from datetime import datetime, timedelta
+      import pytz
+      import random
+
+      from kestra import Kestra
+
+
+      def generate_random_date():
+          start = datetime.now()
+          end = start + timedelta(weeks=1)
+
+          random_date = start + (end - start) * random.random()
+
+          # Set the timezone to +02:00
+          timezone = pytz.FixedOffset(120)  # 120 minutes = 2 hours
+          random_date = random_date.astimezone(timezone)
+
+          # Return formatted date string
+          return random_date.strftime("%Y-%m-%dT%H:%M:%S%z")
+
+
+      # Simulate receiving 10 random scheduled execution dates
+      execution_dates = sorted([generate_random_date() for _ in range(10)])
+
+      Kestra.outputs(dict(execution_dates=execution_dates))
+
+  - id: each
+    type: io.kestra.plugin.core.flow.ForEach
+    values: "{{ outputs.generate_dates.vars.execution_dates }}"
+    concurrencyLimit: 0
+    tasks:
+      - id: subflow
+        type: io.kestra.plugin.core.flow.Subflow
+        namespace: company.team
+        flowId: myflow
+        scheduleDate: "{{ taskrun.value }}"
+        wait: false
+```
+
+
+Finally, you can also use the new `ScheduleOnDates` trigger to start a flow at specific dates known ahead of time. This trigger is useful when you know the exact dates when you want to start the flow:
+
+```yaml
+id: scheduled_at
+namespace: company.team
+
+tasks:
+  - id: print_date
+    type: io.kestra.plugin.core.log.Log
+    message: Hello at {{ trigger.date }}
+
+triggers:
+  - id: schedule
+    type: io.kestra.plugin.core.trigger.ScheduleOnDates
+    timezone: Europe/Berlin
+    recoverMissedSchedules: LAST
+    dates:
+      - 2024-12-24T17:00:00+02:00 # Christmas Eve
+      - 2024-12-25T17:00:00+02:00 # Christmas Day
+      - 2024-12-31T17:00:00+02:00 # New Year's Eve
+      - 2025-01-01T17:00:00+02:00 # New Year's Day
+      - "{{now() | dateAdd(1, 'HOURS')}}"
+      - "{{now() | dateAdd(2, 'DAYS')}}"
+      - "{{now() | dateAdd(3, 'WEEKS')}}"
+      - "{{now() | dateAdd(4, 'MONTHS')}}"
+```
+
+We are excited what you will build with these new Schedule-for-later enhancements! Here is how a [community member reacted](https://github.com/kestra-io/kestra/issues/3818#issuecomment-2330205741) to this feature:
+
+> "This is a game changer for me. I have jobs that need to be run whose schedule time and date can only be derived as a delta from a specific event. This would allow me to calculate the runs for the week, and schedule the jobs that need to run!"
+
+Let us know how you plan to use these scheduling enhancements to make your flows (literally!) future-proof!
+
+## Additional Improvements
```

Comment:
Just sharing that the section title here sounds similar to 'Other Enhancements' to me. Not clear to me why we separate both (other than for our formatting purposes).

Reply:
yup, it is the same but those are too small to have their own sections + would make the post too long. Turned this from H2 to H3 ✅